src/vm/eeconfig.cpp

```diff
@@ -166,6 +166,18 @@ void *EEConfig::operator new(size_t size)
     RETURN g_EEConfigMemory;
 }
 
+BOOL CtrlHandler(DWORD dwEvent)
+{
+#if !CROSSGEN_COMPILE && !FEATURE_PAL
+    if (dwEvent == CTRL_CLOSE_EVENT)
+    {
+        ForceEEShutdown(SCA_ReturnWhenShutdownComplete);
```

Comment:
We pass `SCA_ExitProcessWhenShutdownComplete` on Unix. If the difference is intentional, it maybe worth a comment.

Reply:
The intention here is to allow other handlers to complete. Does Unix not have multiple handlers or are we intentionally blocking them from running?

Reply:
@adityamandaleeka Do you remember why we chose `SCA_ExitProcessWhenShutdownComplete` on Unix?

Reply:
@jkotas It's been a while, so I don't recall all the details about this. I do remember the goal at the time was just to make SIGTERM terminate as fast as possible without doing extra work. I also see that the [comment about ShutdownCompleteAction](https://github.com/dotnet/coreclr/blob/master/src/vm/ceemain.h#L40) makes it sound like `SCA_ReturnWhenShutdownComplete` is only intended for shim-initiated shutdown, so perhaps that also contributed to the decision to use `SCA_ExitProcessWhenShutdownComplete`.